test/SDK.spec.ts

```diff
@@ -44,7 +45,7 @@ describe('SDK - lens contract', () => {
     const signer = wallet.connect(provider)
 
     //Create soulZap object
-    const soulZap = new SoulZap_UniV2_ApeBond(ChainId.POLYGON, signer)
+    const soulZap = new SoulZap_UniV2_ApeBond_SDK(ChainId.POLYGON, signer)
```

Comment:
Naming this _SDK for partners is a bit odd?

Reply:
I agree, I needed to change it though because `SoulZap_UniV2` was colliding with the typechain types. 

Any other suggestions?